ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/workers-rhel/provision/cucushift-installer-rehearse-azure-ipi-workers-rhel-provision-chain.yaml

```diff
@@ -0,0 +1,31 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-workers-rhel-provision
+  steps:
+  - ref: azure-provision-resourcegroup
+  - ref: azure-provision-vnet
+  - chain: azure-provision-bastionhost
+  - ref: ipi-conf
+  - ref: ipi-conf-azure
+  - ref: ipi-conf-azure-provisionednetwork
+  - ref: ipi-install-monitoringpvc
+  - ref: ipi-install-rbac
+  - ref: openshift-cluster-bot-rbac
+  - ref: ipi-install-install
+  - ref: ipi-install-times-collection
+  - ref: workers-rhel-azure-provision
+  - chain: workers-rhel
+  env:
+  - name: RHEL_WORKER_COUNT
+    default: 2
+    documentation: The number of rhel worker to be provisioned.
+  - name: RHEL_VM_SIZE
+    default: "Standard_D4s_v3"
+    documentation: The Size of the RHEL VM provisioned.
+  - name: PLATFORM_VERSION
+    default:  "8.3"
+    documentation: "RHEL node version"
+  - name: RHEL_USER
```

Comment:
I guess we can drop this env from this chain file

Reply:
We can drop it from this can file, because ci-operator/step-registry/workers-rhel/azure-provision/workers-rhel-azure-provision-ref.yaml has set an env var with a default value, right?

Reply:
okay yeah got it👍